<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioManagerForm</class>
 <widget class="QWidget" name="AudioManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Audio</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Musice/Audio</string>
   </property>
  </widget>
  <widget class="QListWidget" name="audioListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Effects</string>
   </property>
   <widget class="QCheckBox" name="FadeInChkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>FadeIn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FadeInTimeLEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="FadeoutChkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>FadeOut</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FadeOutTimeLEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LoopChkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>LoopMusic</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="AddMusicStepPBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>AddMusicStep</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="QueueMusiccheckBox">
      <property name="text">
       <string>AddToQueue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>QueMusic</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="AddMusicPushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>AddMoreMusic</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
